mlearning/cmsis/libcmsisnn/Makefile

```diff
@@ -77,10 +77,6 @@ CSRCS += $(CMSIS5_NN)/ConvolutionFunctions/arm_convolve_HWC_q7_fast_nonsquare.c
 CSRCS += $(CMSIS5_NN)/ConvolutionFunctions/arm_depthwise_conv_s8.c
 CSRCS += $(CMSIS5_NN)/ConvolutionFunctions/arm_convolve_1x1_HWC_q7_fast_nonsquare.c
 CSRCS += $(CMSIS5_NN)/ConvolutionFunctions/arm_convolve_HWC_q7_basic_nonsquare.c
-CSRCS += $(CMSIS5_NN)/ConvolutionFunctions/arm_convolve_CHW_f32_basic_nonsquare.c
-CSRCS += $(CMSIS5_NN)/ConvolutionFunctions/arm_convolve_CHW_q7_basic_nonsquare.c
-CSRCS += $(CMSIS5_NN)/ConvolutionFunctions/arm_convolve_CHW_q15_basic_nonsquare.c
-CSRCS += $(CMSIS5_NN)/ConvolutionFunctions/arm_nn_CHW_mat_mult_kernel_q7_q15.c
```

Comment:
Please revert the change, these files are provided by:
https://github.com/apache/incubator-nuttx-apps/blob/master/mlearning/cmsis/cmsis-nn-support_nnabla.patch

Reply:
Done